aliases/confirm.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+
+// Package aliases implements the validation and signing features of https://ssb-ngi-pointer.github.io/rooms2/#alias
+package aliases
+
+import (
+	"bytes"
+
+	"golang.org/x/crypto/ed25519"
+
+	refs "go.mindeco.de/ssb-refs"
+)
+
+// Registration ties an alias to the ID of the user and the RoomID it should be registerd on
+type Registration struct {
+	Alias  string
+	UserID refs.FeedRef
+	RoomID refs.FeedRef
+}
+
+// Sign takes the public key (belonging to UserID) and returns the signed confirmation
+func (r Registration) Sign(privKey ed25519.PrivateKey) Confirmation {
```

Comment:
I think the server doesn't need this function, because I have this in ssb-room-client, since the signature is created by the client only. Or are you intending that this is used by Go-ssb peers that are clients of the room server?

Reply:
Or is this just for tests? (Would also be decent to have, since it's not that many lines of code)

Reply:
Both? 😅 yea, mostly do be able to run a pure go test client.

Reply:
Ok, no problem here :)